src/handle.rs

```diff
@@ -67,6 +67,14 @@ pub struct AxisMarker {
     pub quaternion: Option<[f32; 4]>,
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct Relationship {
+    pub parent: String,
+    pub child: String,
+    /// Relative position
+    pub position: [f32; 3],
+}
```

Comment:
We need a relative rotation (quaternion).

Reply:
Added quaternion.